prisma/schema.prisma

```diff
@@ -37,6 +37,9 @@ model Group {
   id          String  @id @default(uuid())
   name        String
   description String  @db.Text
+  imgUrl      String?
+  imgAlt      String?
+  socialLinks Json?
```

Comment:
Not sure if we should use Json like this for structured data. I would recommend making these separate fields:

```
twitterUrl: string
linkedinUrl: string
```

Or alternatively, we could split this up into it a second table like:

```
Table Socials {
   url: string
   type?: LinkedIn | Twitter | ...
 }

Table Group_Socials: {
    groupId: String
    socialId: String
}

Table User_Socials: {
    userId: String
    socialId: String
}
```